<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="USER-JKFMHGFFJS" ignored="0" name="Default test" tests="1" failures="1" timestamp="13 十二月 2017 09:31:04 GMT" time="0.181" errors="0">
  <testcase name="f" time="0.0" classname="com.testapi.cas.NewTest">
    <failure type="org.testng.TestNGException" message="
Data Provider public java.lang.Object[][] com.testapi.cas.NewTest.createData() throws java.io.IOException returned a null value">
      <![CDATA[org.testng.TestNGException: Data Provider public java.lang.Object[][] com.testapi.cas.NewTest.createData() throws java.io.IOException returned a null valueat org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:133)at org.testng.internal.Parameters.handleParameters(Parameters.java:706)at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:914)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- f -->
</testsuite> <!-- Default test -->
